hashes/contrib/test.sh

```diff
@@ -52,8 +58,12 @@ if [ "$DO_FEATURE_MATRIX" = true ]; then
     cargo test --no-default-features --features="std,schemars"
 fi
 
+REPO_DIR=$(git rev-parse --show-toplevel)
+
 if [ "$DO_SCHEMARS_TESTS" = true ]; then
-    (cd extended_tests/schemars && cargo test)
+    pushd "$REPO_DIR/hashes/extended_tests/schemars" > /dev/null
+    cargo test
+    popd
```

Comment:
In d02a984a1906fd5175e0e7eed7e54e0fed2d554f:

May want to redirect `popd` to null as well if you're redirecting `pushd`.

Reply:
I added the redirect to null, FTR on my system `popd` does not output anything anyways?

Reply:
Interesting. On my system `popd` also outputs the directory that it's changing to. I have bash 5.1. Maybe this is an obscure setting somewhere.

Reply:
derrrr me no Linux - I'm using zsh as my shell.